packages/css-data/src/custom-data.ts

```diff
@@ -18,9 +18,16 @@ export const valueTypes = [
   "url",
   "string",
   "custom-ident",
+  "dashed-ident",
   "single-animation-composition",
   "palette-identifier",
   "flex",
+  "inset-area",
+  "offset-path",
+  "coord-box",
+  "anchor-element",
+  "try-tactic",
+  "try-size",
```

Comment:
I'm not sure if there is any use in these types

Reply:
I think the question is rather: would it hurt to support as much as possible?

Reply:
We are saying we support full css on the site

Reply:
Those are not properties but css types

Reply:
values, properties, all part of css, right?

Reply:
I mean specifically this list is used only to detect custom idents